main.py

```diff
@@ -1,45 +1,42 @@
-import requests
 import json
 from flask import Flask, request
-app = Flask(__name__)
-
-REQUIRED_LABELS = ['ready for review']
+from PullRequest import PullRequest
```

Comment:
module names in python are normally snake_cased. so that file should be called `pull_request.py`. However, we normally don't put single classes in a file, so I would probably have called that file `models.py` or `utils.py` or something similar.

Reply:
ah cool didn't know that. that is very much the opposite of Java